airbyte-scheduler/Dockerfile

```diff
@@ -25,6 +25,15 @@ WORKDIR /app
 ADD https://github.com/ufoscout/docker-compose-wait/releases/download/${WAIT_VERSION}/wait wait
 RUN chmod +x wait
 
+# Install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/linux/amd64/kubectl
```

Comment:
this seems reasonable for now, but should always be publishing the scheduler such that it includes both docker and k8s? or would the goal in the future be to split the artifacts?

Reply:
Long term splitting might make more sense, especially if we need customizations for other environments. For now this just seems easier.